source/common/network/cidr_range.h

```diff
@@ -137,6 +137,7 @@ class IpList {
 
   bool contains(const Instance& address) const;
   size_t getIpListSize() const { return ip_list_.size(); };
+  bool isEmpty() const;
```

Comment:
`getIpListSize()` was recently added for the exact same purpose. Let's use one or the other; I don't care which. So please either use `getIpListSize() == 0` in this PR, or replace the use of `getIpListSize() == 0` with this new function.

Also, I prefer the name `empty()` to `isEmpty()` because it lines of with stl containers.

Reply:
i've removed my changes and stuck with using `getIpListSize`.